pygsp/tests/test_filters.py

```diff
@@ -19,8 +19,8 @@ class TestCase(unittest.TestCase):
     def setUpClass(cls):
         cls._G = graphs.Sensor(123, seed=42)
         cls._G.compute_fourier_basis()
-        cls._rs = np.random.RandomState(42)
-        cls._signal = cls._rs.uniform(size=cls._G.N)
+        cls._rng = np.random.default_rng(42)
```

Comment:
Here again, but since it is a test, it is less an issue.

Reply:
Tests and docstrings are kinda similar, since we execute the docstrings and test their outputs.